### PR TITLE
Tomcat 8.0.35+ support

### LINKED ISSUE #5: Tomcat 6 dependency in 8 module
It appears the core module depends on Tomcat 6, which has an `attributes` field in `org.apache.catalina.session.StandardSession`, but that field is gone in Tomcat 8, leading to this:
```
java.lang.NoSuchFieldError: attributes
       at com.hazelcast.session.HazelcastSession.writeData(HazelcastSession.java:85)
```

### PR DESCRIPTION
**Pull request by emrahkocaman**
fixes #5

### CODE REVIEW

**Comment by mesutcelik** [comment]
👍

File: tomcat-core/src/main/java/com/hazelcast/session/HazelcastSession.java [hunk 1]
```diff
@@ -95,11 +93,41 @@ public void readData(ObjectDataInput objectDataInput) throws IOException {
...
+     */
+    private Field getAttributesField() {
+        try {
+            Field attributesField = StandardSession.class.getDeclaredField("attributes");
+            attributesField.setAccessible(true);
+            return attributesField;
+        } catch (NoSuchFieldException e) {
+            throw new RuntimeException(e);
+        }
+    }
```
Comment (nilskp) [inline tomcat-core/src/main/java/com/hazelcast/session/HazelcastSession.java]: Seems like we could do this statically or lazily, to avoid full reflection impact on every single call.

Comment (emrahkocaman) [inline tomcat-core/src/main/java/com/hazelcast/session/HazelcastSession.java]: Thanks @nilskp , I've sent another PR for that https://github.com/hazelcast/hazelcast-tomcat-sessionmanager/pull/7
